src/frontends/tensorflow/tests/test_models/models_pbtxt/partitioned_call2.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import tensorflow.compat.v1 as tf
+
```

Comment:
is it necessary to use double line breaks between blocks? why should it mean?

Reply:
this is PyCharm formatting that we follow to.